### ISSUE TITLE
Windows testing

### ISSUE THREAD

**Issue by jayvdb** [pglass/tox-pip-version#14]
.. either with AppVeyor or Azure, etc.

In my experience this plugin doesnt work so well on Windows, often due to problems with pip uninstalling itself on Windows.  It would be useful to have a few scenarios running on a Windows CI to ensure it works.  My working setup is preinstalling pip to a certain version before running tox, and using tox-venv and `sitepackages=true` with tox-pip-version configured to use the exact same pip version as pre-installed.  The result is effectively a very complicated no-op.

I can help with AppVeyor, but not Azure.

**Comment by pglass** [comment]
Yeah, haven't tested at all outside of Unix. Poking around the internet, there is a way to test python/windows on travis, e.g. https://github.com/WeblateOrg/translation-finder/blob/master/.travis.yml#L20-L26 so would like to try that first, although per [the travis docs](https://docs.travis-ci.com/user/reference/windows/) only "Windows Server, version 1803 is currently supported"

Any particular python/windows versions you're interested in (sounds like 3.4 or 3.6 from other comments)?

**Comment by jayvdb** [comment]
I havent tried Windows on Travis.  It looks good.  Ya, py34,36 is as much Windows pain as I can handle these days.

**Comment by jayvdb** [comment]
Giving it a whirl on a rather large matrix .. https://travis-ci.org/jayvdb/coala-bears/builds/545114921

main Python jobs fail on
> The language 'python' is currently unsupported on the Windows Build Environment.

Easy to workaround of course, but means the .travis.yml for Linux + Windows is going to be a bit messy for the setup.

One important aspect is the jobs started very quickly, unlike the Travis' OSX jobs, so it could be worth the pain of setting it up.

**Comment by jayvdb** [comment]
I've found it to be quite good, but a bit brittle, so can require a bit more Windows knowledge than AppVeyor to get working nicely.

https://github.com/cclauss/itinerant-tester-windows is a simple rig for multiple python versions, found courtesy of https://travis-ci.community/t/pip-in-python-on-windows/1720/3 .

https://travis-ci.org/jayvdb/coala-bears/jobs/547079066 is a horribly complicated rig for a single python version ;-)

**Comment by jayvdb** [comment]
See https://github.com/tox-dev/tox-travis/issues/133#issuecomment-518634649 for a very basic job definition for Travis Windows which uses tox-travis.
